force-app/main/default/permissionsets/Mainly_Digital_Manager.permissionset-meta.xml

```diff
@@ -77,10 +77,6 @@
         <tab>Journey__c</tab>
         <visibility>Visible</visibility>
     </tabSettings>
-    <tabSettings>
```

Comment:
I see what you've done here and it works. Do we need the tabs at all in the application? Is it a matter of visibility?